installer/lib/messages.py

```diff
@@ -90,7 +90,7 @@ def dest_path(dest=None) -> Path:
         dest = Path(dest).expanduser().resolve()
     else:
         dest = Path.cwd().expanduser().resolve()
-    prev_dest = dest.expanduser().resolve()
+    prev_dest = init_path = dest
```

Comment:
I can't quite figure out what is going on here.

Reply:
Yeah, it's a bit less than obvious. It's two things. I'm removing an unnecessary extra call to `.expanduser().resolve()` since this was already done on `dest` and then I am also assigning the value to `init_path` which is used below but never assigned.